HTML5-frontend/script.js

```diff
@@ -638,21 +587,13 @@ function checkInstalledPlugins() {
   } catch(e) {
     hasWebsockets = false;
   }
-
-  if (!hasWebsockets) {
-    $('#websocketButton').attr('disabled', true);
-  }
-
-  if (!hasJava) {
-    $('#javaButton').attr('disabled', true);
-  }
-
   if (hasWebsockets) {
     useWebsocketAsBackend();
```

Comment:
I think if you move the useWebsocketAsBackend function inline here, it will improve readability